python/composio/client/http.py

```diff
@@ -18,59 +16,8 @@
 DEFAULT_REQUEST_TIMEOUT = 60.0
 
 
-class AsyncHttpClient(AsyncSession, logging.WithLogger):
-    """Async HTTP client for Composio"""
-
-    def __init__(
-        self,
-        base_url: str,
-        api_key: str,
-        runtime: t.Optional[str] = None,
-        loop: t.Optional[AbstractEventLoop] = None,
-        timeout: t.Optional[float] = None,
-    ) -> None:
-        """
-        Initialize async client channel for Composio API
-
-        :param base_url: Base URL for Composio API
-        :param api_key: API key for Composio API
-        :param runtime: Runtime specifier
-        :param timeout: Request timeout
-        :param loop: Event for execution requests
-        """
-        AsyncSession.__init__(
-            self,
-            loop=loop,
-            headers={
-                "x-api-key": api_key,
-                "x-source": SOURCE_HEADER,
-                "x-runtime": runtime or DEFAULT_RUNTIME,
-            },
-        )
-        logging.WithLogger.__init__(self)
-        self.base_url = base_url
-        self._request_timeout = timeout or DEFAULT_REQUEST_TIMEOUT
-
-    def _wrap(self, method: t.Callable) -> t.Callable:
-        """Wrap http request."""
-
-        def request(url: str, **kwargs: t.Any) -> t.Any:
-            """Perform HTTP request."""
-            self._logger.debug(
-                f"{method.__name__.upper()} {self.base_url}{url} - {kwargs}"
-            )
-            return method(url=f"{self.base_url}{url}", **kwargs)
-
-        return request
-
-    def __getattribute__(self, name: str) -> t.Any:
-        if name in ("get", "post", "put", "delete", "patch"):
-            return self._wrap(super().__getattribute__(name))
-        return super().__getattribute__(name)
-
-
 class HttpClient(SyncSession, logging.WithLogger):
-    """Async HTTP client for Composio"""
+    """HTTP client for Composio"""
```

Comment:
The updated docstring for HttpClient is more accurate now. Good attention to detail! Consider adding more detailed comments or docstrings to the HttpClient class, especially if it's now the primary HTTP client for the project. This could help future developers understand its usage and importance.